util/cron/nightly

```diff
@@ -710,6 +714,14 @@ if ($runtests == 0) {
     }
 
     $ENV{'CHPL_HOME'} = $chplhomedir;
+
+    # Confirm Chapel built correctly before start_test / paratest
+    if ($buildcheck == 1) {
+        $buildcheckcommand = "cd $ENV{'CHPL_HOME'} && make check";
+        $buildcheckstatus = mysystem($buildcheckcommand, "Running `make check`", 1, 1, 1);
+        print "$buildcheckstatus";
```

Comment:
I think mysystem will print out out the status code, might want to double check that though